### DISCUSSION TITLE
autoscaling/tke-autoscaling-placeholder

### DISCUSSION THREAD

**Discussion by giscus** [imroc/tke-guide#15] [General]
# autoscaling/tke-autoscaling-placeholder

操作场景

https://imroc.cc/tke/autoscaling/tke-autoscaling-placeholder

**Comment by ermazi** [comment]
这块貌似只考虑了新负载的情况; 如果是线上已经有大量负载在跑的情况, 意思是把所有负载重建, 修改为更高优先级吗?
placeholder的占位优先级是-1, 高优1000000, 默认 0; 如果 placeholder 足够的情况, 那么抢占 placeholder 没问题, 如果不够, 是不是会抢占默认优先级 pod呢?

**Reply by imroc** [reply]
应该也有可能, 这种抢占方式实际生产上用的不多, 不如用超级节点

**Reply by ermazi** [reply]
超级节点最大的问题是按照 limit 来申请, 一些没有明确负载边界的应用, 就比较难搞
